django/core/management/templates.py

```diff
@@ -345,6 +346,16 @@ def is_url(self, template):
         scheme = template.split(':', 1)[0].lower()
         return scheme in self.url_schemes
 
+    def apply_umask(self, old_path, new_path):
+        """Apply umask if operating system supports it."""
+        if not os.name == 'nt':
```

Comment:
Should we check `posix` instead?
```suggestion
        if os.name == 'posix':
```

Reply:
Agree, `os.name == 'posix'` makes sense and happy to change; in practice there seems to be no precedence in the code base for that with most exceptions being based on `sys.platform == 'win32'`.